SPECS/util-linux/util-linux.spec

```diff
@@ -107,12 +111,21 @@ rm -rf %{buildroot}/lib/systemd/system
 
 %files devel
 %defattr(-,root,root)
+%license Documentation/licenses/COPYING.LGPL-2.1-or-later libsmartcols/COPYING
+%license libblkid/COPYING
 %{_libdir}/pkgconfig/*.pc
 %{_libdir}/*.so
+/lib/libblkid.so.*
+/lib/libsmartcols.so.*
```

Comment:
.so.* files should be in the libs package instead of devel.

They should also be using a macro for libdir but I don't consider that a merge blocker.